scss/editor/_layout.scss

```diff
@@ -314,15 +315,68 @@
             .k-upload-button {
                 margin: 0;
             }
+
+            .k-upload-status {
+                display: none;
+            }
         }
 
+        .k-upload-files {
+            display: block;
+        }
+
+        // Tiles
         .k-tiles {
             height: 390px;
             margin: 0 0 $spacer-y 0;
             padding: $padding-y $padding-x;
             border-top-width: 0;
             overflow: auto;
         }
+
+        .k-tile {
+            float: left;
+            width: 33%;
+            height: 90px;
+            padding: $padding-y $padding-x;
```

Comment:
This could be improved:

```scss
.k-tiles {
    // current styles...
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
}
```

The same applies for the content of `.k-tile`